freqtrade/exchange/exchange.py

```diff
@@ -472,26 +472,31 @@ def dry_run_order(self, pair: str, ordertype: str, side: str, amount: float,
             'pair': pair,
             'price': rate,
             'amount': _amount,
-            "cost": _amount * rate,
+            'cost': _amount * rate,
             'type': ordertype,
             'side': side,
             'remaining': _amount,
             'datetime': arrow.utcnow().isoformat(),
             'status': "closed" if ordertype == "market" else "open",
             'fee': None,
-            "info": {}
+            'info': {}
         }
-        self._store_dry_order(dry_order)
+        self._store_dry_order(dry_order, pair)
         # Copy order and close it - so the returned order is open unless it's a market order
         return dry_order
 
-    def _store_dry_order(self, dry_order: Dict) -> None:
+    def _store_dry_order(self, dry_order: Dict, pair: str) -> None:
         closed_order = dry_order.copy()
-        if closed_order["type"] in ["market", "limit"]:
+        if closed_order['type'] in ["market", "limit"]:
```

Comment:
* should "stop-loss" (market stoploss for Kraken) be also added here, to simulate it as fulfilled immediately?
* should "limit" be left open, to be closed at next iteration?

Reply:
Stoploss handling for this is "a few" lines below (stoploss orders do NOT close immediately when they are placed!)

limit IS left open until the next iteration (see line 480).

Reply:
It only handles "stop_loss_limit" a few lines below, for binance. Kraken has another order-type set (and market stoplosses)

Reply:
> limit is left open until the next iteration (see line 480).

ah, got it, the closed_order is "closed" and dry_order is either open or closed depending on ordertype...
👍 
same seems to be for stoplosses (open in dry_order), okay